messages/deploy.md

```diff
@@ -18,12 +18,20 @@ Use this command for quick and simple deploys. For more complicated deployments,
 
 - Deploy a project and use stored values from a previous command run:
 
-     <%= config.bin %> <%= command.id %>
+  <%= config.bin %> <%= command.id %>
 
 - Reprompt for all deployment inputs:
 
-     <%= config.bin %> <%= command.id %> --interactive
+  <%= config.bin %> <%= command.id %> --interactive
 
 # flags.interactive.summary
 
 Force the CLI to prompt for all deployment inputs.
+
+# error.initialization
+
+One or more initialization steps failed.
+
+# error.initialization.title
+
+Initialization Step Failures - The following table describes each failure
```

Comment:
Sorry this is late...

I'm being really nitpicky here, but something about your suggested punctuation is bugging me.  And adding "Step" to the title is confusing, IMHO.  But ugh, I don't know...  How about this:

Initialization Failures. The following table describes each failure:

Reply:
@peternhale ^